apps/condo/domains/billing/utils/clientSchema/BillingCategory.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createschema billing.BillingCategory 'name:Text'`
+ */
+
+import { pick, get } from 'lodash'
+
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import { generateReactHooks } from '@condo/domains/common/utils/codegeneration/generate.hooks'
+
+import { BillingCategory as BillingCategoryGQL } from '@condo/domains/billing/gql'
+import { BillingCategory, BillingCategoryUpdateInput, QueryAllBillingCategoriesArgs } from '../../../../schema'
```

Comment:
Could u plz change schema imports to absolute path?

Reply:
Ok